tests/test_execution.py

```diff
@@ -412,8 +356,7 @@ def test_aggregate_composite_traintuples(global_execution_env):
     # username/password are not available in the settings files.
 
     client = clients[0]
-    initial_assets = initial_assets.filter_by(client.node_id)
-    dataset = initial_assets.datasets[0]
+    dataset = default_datasets.sort_by(client.node_id)
```

Comment:
I don't think this is working as `default_datasets` is a list and a list instance has no method `sort_by`.
Could be replaced maybe with `dataset = default_datasets[0]`?

Reply:
You're right, I'll do that!